tests/runner/Testcontracts.ml

```diff
@@ -115,7 +115,7 @@ let rec build_contract_tests_with_init_file env name exit_code i n
           (* if the test is supposed to succeed we read the output from a file,
                  otherwise we read from the output stream *)
           let interpreter_output =
-            if exit_code = succ_code then In_channel.read_all output_file
+            if Poly.(exit_code = succ_code) then In_channel.read_all output_file
```

Comment:
Does this need to be polymorphic? Don't we know the type of `succ_code`?

Reply:
The type is `Unix.process_status`. AFAIK, it does not have a special equality function. I'm not sure if it can be generated as we don't have access to the source code.

Reply:
Got it. Let's just keep it as it is.